packages/cadl-ranch-specs/http/server/versions/not-versioned/main.tsp

```diff
@@ -0,0 +1,38 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+
+/**
+ * Illustrates not-versioned server.
```

Comment:
What is the scenario for these two cases? A service that accept api version param, but not defined the potential value? How should emitter deal with the api version param?

Reply:
See the description.